pkg/sidecar/k8s_reactor.go

```diff
@@ -37,26 +37,37 @@ var (
 )
 
 type K8sReactor struct {
-	client         *sync.Client
-	manager        *docker.Manager
-	servicesRoutes []net.IP
+	client          *sync.Client
+	manager         *docker.Manager
+	allowedServices []AllowedService
 }
 
 func NewK8sReactor() (Reactor, error) {
-	// TODO: Generalize this to a list of services.
-	wantedRoutes := []string{
-		os.Getenv(EnvRedisHost),
-		os.Getenv(EnvInfluxdbHost),
+	wantedServices := []struct {
+		name string
+		host string
+	}{
+		{
+			"redis",
+			os.Getenv(EnvRedisHost),
+		},
+		{
+			"influxdb",
+			os.Getenv(EnvInfluxdbHost),
+		},
 	}
 
-	var resolvedRoutes []net.IP
-	for _, route := range wantedRoutes {
-		ip, err := net.ResolveIPAddr("ip4", route)
+	var resolvedServices []AllowedService
+	for _, s := range wantedServices {
+		if s.host == "" {
```

Comment:
This is a bug currently on `master` - we are trying to resolve an empty host (`influxdb`).

Reply:
👍